utils.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+var _ = _ || (typeof require === 'function' && require('lodash'));
+
```

Comment:
@odino about your comment

> [can we do it through webpack? I assume if you webpack it and _ is redefined it wont be a problem](https://github.com/p16/fdpn/pull/1#discussion_r74370906)

I'll check that and see if we can do it.

BTW: I couldn't install reconfig 3.0.0 because of some problems using the [`build` files](https://github.com/namshi/reconfig/tree/master/build), I need to check that as well.